src/main/java/laledevri/laladevri/App.java

```diff
@@ -8,6 +8,20 @@
 {
     public static void main( String[] args )
     {
+
+int target2 = -5;
+
+
+
+	int num3 = 3;
+
+
+
+final int deneme2;
+
+
+
+	target2 =- num3;  // Noncompliant; target = -3. Is that really what's meant?
         System.out.println( "Hello World!" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)